mino/router/mod.go

```diff
@@ -67,6 +67,9 @@ type Router interface {
 // Routes is a set of relay addresses where to send the packet.
 type Routes map[mino.Address]Packet
 
+// Voids is a set of addresses that cannot be addressed by the routing table.
+type Voids map[mino.Address]error
```

Comment:
One day it could be useful to store more information than a simple error, which could help find a better path. (it could tell if its a timeout, authentication failure, bad acting, too-slow, etc..)